app/src/main/java/chat/rocket/android/helper/RxHelper.java

```diff
@@ -0,0 +1,23 @@
+package chat.rocket.android.helper;
+
+import java.util.concurrent.TimeUnit;
+import rx.Observable;
+import rx.Single;
+import rx.functions.Func0;
+import rx.functions.Func1;
+
+/**
+ * Rx operator and so on.
+ */
+public class RxHelper {
+  public static Func1<Observable<? extends Throwable>, Observable<?>> exponentialBackoff(
+      int maxRetryCount, long base, TimeUnit unit) {
+    return errors -> errors
+        .zipWith(Observable.range(0, maxRetryCount), (error, retryCount) -> retryCount)
```

Comment:
Actually I made this using [javadoc](https://github.com/ReactiveX/RxJava/blob/a8ba158839b67246a742b6f1531995ffd7545c08/src/main/java/io/reactivex/Observable.java#L9601) as reference, and didn't know the detail...
I will study it...